spec/feature/place_spec.rb

```diff
@@ -0,0 +1,240 @@
+require "gds_api/test_helpers/places_manager"
+
+RSpec.describe "Places", type: :feature do
+  include GdsApi::TestHelpers::PlacesManager
+
+  before do
+    @payload = { title: "Find a passport interview office", base_path: "/passport-interview-office", schema_name: "place", document_type: "place", phase: "beta", public_updated_at: "2012-10-02T15:21:03+00:00", details: { introduction: "<p>Enter your postcode to find a passport interview office near you.</p>", more_information: "Some more info on passport offices", need_to_know: "<ul><li>Proof of identification required</li></ul>", place_type: "find-passport-offices" }, external_related_links: [] }
+    stub_content_store_has_item("/passport-interview-office", @payload)
+    @places = [{ "access_notes" => "The London Passport Office is fully accessible to wheelchair users. ", "address1" => nil, "address2" => "89 Eccleston Square", "email" => nil, "fax" => nil, "general_notes" => "Monday to Saturday 8.00am - 6.00pm. ", "location" => { "longitude" => -0.14411606838362725, "latitude" => 51.49338734529598 }, "name" => "London IPS Office", "phone" => "0800 123 4567", "postcode" => "SW1V 1PN", "text_phone" => nil, "town" => "London", "url" => "http://www.example.com/london_ips_office" }, { "access_notes" => "The doors are always locked.\n\nAnd they are at the top of large staircases.", "address1" => nil, "address2" => "Station Way", "email" => nil, "fax" => nil, "general_notes" => "Monday to Saturday 8.00am - 6.00pm.\n\nSunday 1pm - 2pm.", "location" => { "longitude" => -0.18832238262617113, "latitude" => 51.112777245292826 }, "name" => "Crawley IPS Office", "phone" => nil, "postcode" => "RH10 1HU", "text_phone" => nil, "town" => "Crawley", "url" => nil }]
+  end
+
+  context "when visiting the start page" do
+    before { visit "/passport-interview-office" }
```

Comment:
I think the spacing has been lost in the conversion in this file

Reply:
Fixed